CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing to CC Open Source
+
+Thank you for your interest in contributing to CC Open Source! This document is a set of guidelines to help you contribute to this project.
+
+## Code of Conduct
+
+By participating in this project, you are expected to uphold our [Code of Conduct](https://creativecommons.github.io/community/code-of-conduct/).
+
+## Project Documentation
+
+Please consult the `README` if there is one, it should contain or link to project documentation. If there is no `README`, please file a GitHub issue to create one.
```

Comment:
Could you update this section to point to the actual project documentation? I know this project has a `README` so a bunch of this text is unnecessary.

Reply:
Fixed